content/terms/premium-support.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: markdown
+title: Terms of Service for Sourcegraph Premium Support
+---
+
+Last Modified: May 5, 2023
+
+This document outlines the basic terms and conditions that apply to Sourcegraph's Premium Support Offerings.
+
+## Premium Support
+
+### Benefits of Premium Support
+Purchasing Premium Support provides the following services beyond those provided by Sourcegraph's standard support model:
+- **Accelerated Service Level Agreements (SLAs)** - Accelerated triaging, faster initial response times, and queue priority lead reduced times to resolution.
```

Comment:
This is true except the faster initial response times. The times we hold ourselves to will remain the same but we will bump the tickets to top of the queue which will mean faster time for someone to pick it up.